_posts/2019-08-08-Neue-Stable-Version.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "Endlich eine neue Stable-Version"
+author: oliver
+date:   2019-08-08 18:12:27 +0200
+---
+
+Es hat lange gedauert, aber nun gibt es endlich wieder eine neue Stable-Firmware: beim Freifunk-Treffen wurde beschlossen, dass die derzeitige Testing-Firmware [2018.2.1+bremen1](https://wiki.ffhb.de/Firmware/Changelog#freifunk-bremen-versionen_2018-2-1-bremen1) lange genug im Test war und somit endlich im ganzen Bremer Netz verwendet werden soll.
+
+Gegen die breite Veröffentlichtung dieser Version hatte zuletzt nur noch gesprochen, dass einige Geräte wie z.B. der Archer C7 damit keine Mesh-Verbindungen im 5GHz-Band aufbauen können. Im Laufe der [Problemanalyse](https://github.com/freifunk-gluon/gluon/issues/1584) hat sich dann aber herausgestellt, dass dieses Problem schon in der derzeitigen Stable-Version (der altehrwürdigen 2017.1.8+bremen1) vorhanden war, so dass ein Upgrade zumindest keine existierenden Mesh-Verbindungen unterbrechen dürfte.
+
+## Neuerungen
+
+Seit der [letzten Stable-Version](https://wiki.ffhb.de/Firmware/Changelog#freifunk-bremen-versionen_2017-1-8-bremen1) ist schon fast ein ganzes Jahr vergangen, in dem die Entwickler von OpenWRT, LEDE, Gluon und anderen Projekten viel getan haben. Neben etlichen Verbesserungen am Unterbau dürfte die Unterstützung für diverse neue Geräte die merklichste Änderung sein: TP-Link Archer C7 v5 und CPE210 v2, AVM Fritz!Box 4020 und Fritz!WLAN Repeater 300E und 450E, sowie Ubiquiti UniFi AC Mesh Pro sind nur einige der neu unterstützten Geräte.  
+
+Zu den Änderungen am Unterbau gehört unter anderem der Wechsel von LEDE zurück zu OpenWRT (nachdem diese beiden Projekte wieder zusammengeführt wurden) sowie viele Fehlerbehebungen. Eine detailliertere Liste der Änderungen findet ihr wie immer im [Changelog](https://wiki.ffhb.de/Firmware/Changelog).
```

Comment:
```suggestion
Zu den Änderungen am Unterbau gehört unter anderem der Wechsel von LEDE zurück zu OpenWRT (nachdem diese beiden Projekte wieder zusammengeführt wurden) sowie viele Fehlerbehebungen.
Eine detailliertere Liste der Änderungen findet ihr wie immer im [Changelog](https://wiki.bremen.freifunk.net/Firmware/Changelog).
```

Set URL to _bremen.freifunk.net_ + split a sentence per Line, for easier review in future.

(Markdown rendering ignore single linebreaks)